charts/langsmith/values.yaml

```diff
@@ -14,27 +14,27 @@ images:
   imagePullSecrets: []
   backendImage:
     repository: "docker.io/langchain/langchainplus-backend"
-    pullPolicy: Always
-    tag: "latest"
+    pullPolicy: IfNotPresent
+    tag: "99d8f59"
   frontendImage:
     repository: "docker.io/langchain/langchainplus-frontend-dynamic"
-    pullPolicy: Always
-    tag: "latest"
+    pullPolicy: IfNotPresent
+    tag: "99d8f59"
   hubBackendImage:
     repository: "docker.io/langchain/langchainhub-backend"
-    pullPolicy: Always
-    tag: "latest"
+    pullPolicy: IfNotPresent
+    tag: "99d8f59"
   playgroundImage:
     repository: "docker.io/langchain/langchainplus-playground"
-    pullPolicy: Always
-    tag: "latest"
+    pullPolicy: IfNotPresent
+    tag: "99d8f59"
```

Comment:
use static tags so that people don't accidentally pull in changes they aren't ready for